ui/src/app/job-details/job-details.component.html

```diff
@@ -1,4 +1,4 @@
 <jm-header [showControls]="false"></jm-header>
 <jm-panels [job]="job"></jm-panels>
 <!-- Only show the tasks table if there is at least 1 task. -->
-<jm-tasks *ngIf="hasTasks()" [tasks]="job.tasks" [jobId]="job.id"></jm-tasks>
+<jm-tasks *ngIf="hasTasks()" [tasks]="job.extensions.tasks" [job]="job"></jm-tasks>
```

Comment:
We could consider just passing in `job` here since the tasks are available on the job. But that doesn't need to happen in this PR since it's already such a large change!

Reply:
Hmm that's a good point, I think that would work but mind if I leave that to another PR?

Reply:
I think that's fine!